docs/getting-started/golden-path.md

```diff
@@ -0,0 +1,32 @@
+# The Golden Path
+
+Welcome to the world of Decred! We know that getting started with any new cryptocurrency
+can be difficult and we want to make the journey as easy as possible. Decred is fully
+documented, with everything from basic information such as, "how do I send decred", to
+super technical stuff like "why did you choose a BLAKE-256 hash function, and how does
+it compare to the Merkle-Damgård construction?".
+
+So that you don't get overwhelmed by the amount of information available, we have created
+a "Golden Path" through the documentation that will take you through the basics of getting
+started with Decred, information on how to participate in the voting and mining processes
+then finish with advanced options and how to participate in the test network for those
+who are interested. 
+
+Each document in the path will lead to the next so have a look at the bottom of the page
+for the 'Continue to ...' link to keep going.
+
+Continue to download and install...  
+***WINDOWS***
+
+* [Paymetheus](/getting-started/install-guide#windows-installer)
```

Comment:
I brought this up in the documentation slack channel. Issue #92 shows the existing CSS code that is making lists within the actual page div have a font-weight @ 600. I believe @gozart1 has something in PR #81 to fix it.